src/ui/components/SecondaryToolbox/react-devtools/__tests__/utils.ts

```diff
@@ -1,5 +1,5 @@
+import type { Store } from "@replayio/react-devtools-inline";
 import { createElement } from "react";
-import type { Store } from "react-devtools-inline";
```

Comment:
We could add an entry to the `no-restricted-imports` rule for `"react-devtools-inline"` but I guess it's probably not necessary